committed.toml

```diff
@@ -14,3 +14,6 @@ subject_not_punctuated = true
 line_length = 0
 # disable subject length
 subject_length = 0
+
```

Comment:
nit: maybe remove the newline for consistency

Reply:
Done